src/msw/fontdlg.cpp

```diff
@@ -126,21 +126,22 @@ int wxFontDialog::ShowModal()
         chooseFontStruct.lpfnHook = wxFontDialogHookProc;
     }
 
+    // The native font chooser does not support moving between displays with
+    // different DPIs. Check if it will be shown system-dpi-aware.
+    const bool useSystemDPI = wxMSWImpl::AutoSystemDpiAware::Needed();
+
+    // When the font chooser is system-dpi-aware, it expects the font at 96DPI/100% scaling.
+    // When the font chooser is per-monitor-dpi-aware, it expects a font with the system DPI.
+    const int chooserDPI = useSystemDPI ? wxDisplay::GetStdPPIValue() : wxGetDPIofHDC(ScreenHDC()).y;
```

Comment:
This is very minor, but I thought this was going to be used to "choose" something. I'd call this just `dialogDPI` or `fontdlgDPI`.